protoc-artifacts/README.md

```diff
@@ -54,7 +54,62 @@ Before you can upload artifacts to Maven Central repository, make sure you have
 read [this page](http://central.sonatype.org/pages/apache-maven.html) on how to
 configure GPG and Sonatype account.
 
-Use the following command to upload artifacts:
+You need to perform the deployment for every platform that you want to
+suppport. DO NOT close the staging repository until you have done the
```

Comment:
I just added the options for cross-compiling for Windows on Linux. Unfortunately MinGW for Linux will add ``libwinpthread-1.dll`` dependency which cannot be removed. So we still have to build Windows artifacts on Windows.